src/api/ranking/ranking.types.ts

```diff
@@ -1,3 +1,14 @@
 export enum Goal {
   GOAL = 2222,
 }
+
+export type UserDestails = {
```

Comment:
Czy to są na pewno szczegóły użytkownika? Może bardziej statystyki zawodnika? ;)